src/plugins/intel_cpu/thirdparty/CMakeLists_libxsmm.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Maybe I missed something... But seems like we don't need this file because you install this lib in the common CMakeLists file. Could you please check this moment?

Reply:
You're right, we don't need it